src/components/Survey.vue

```diff
@@ -193,10 +206,33 @@ export default {
           console.log("Error loading Questionnaire ", e);
         });
     },
+    async setCarePlan() {
+      const carePlan = await getPatientCarePlan(
+        this.client,
+        this.patient.id
+      ).catch((e) => console.log("Get care plan error ", e));
```

Comment:
if this is just debugging info, no worries, but it would be of value to add error logging here and elsewhere to capture client problems.